modules/riemann/riemann-parser.c

```diff
@@ -48,6 +48,7 @@ static CfgLexerKeyword riemann_keywords[] =
   { "ca_file",                  KW_CA_FILE },
   { "cert_file",                KW_CERT_FILE },
   { "key_file",                 KW_KEY_FILE },
+  { "flush_lines",              KW_BATCH_LINES, KWS_OBSOLETE, "The flush-lines option is deprecated in favour of batch-lines()" },
 
```

Comment:
again, we probably don't need this as the cfg-parser.c definition prevails.

Reply:
The reason why I've added this to x-parser.c is to mark it as an obsolete option and recommend a new option name.
In cfg-parser it is not obsolete (and don't want to make it obsolete currently, and even if it would be obsolete, the recommended option name would be different one... so I think we need this change).